test/sql/bgw_launcher.sql

```diff
@@ -148,7 +148,21 @@ DROP EXTENSION timescaledb;
 COMMIT;
 SELECT wait_worker_counts(1,0,0,0);
 
-/* Connect to the template1 database */ 
+/* Test that background workers are stopped with DROP OWNED */
+ALTER ROLE :ROLE_DEFAULT_PERM_USER WITH SUPERUSER;
+\c single_2  :ROLE_DEFAULT_PERM_USER
+SET client_min_messages = ERROR;
+CREATE EXTENSION timescaledb CASCADE;
+RESET client_min_messages;
+/* Make sure there is 1 launcher and 1 bgw in single_2 */
+SELECT wait_worker_counts(launcher_ct=>1, scheduler1_ct=> 0, scheduler2_ct=>1, template1_ct=>0);
+DROP OWNED BY  :ROLE_DEFAULT_PERM_USER;
+/* The worker in single_2 is dead. Note that 0s are respected */
+SELECT wait_worker_counts(launcher_ct=>1, scheduler1_ct=>0, scheduler2_ct=>0, template1_ct=>0);
+\c single_2  :ROLE_SUPERUSER
+ALTER ROLE :ROLE_DEFAULT_PERM_USER WITH NOSUPERUSER;
```

Comment:
Can you add a test where `drop owned by` passes a user that does NOT have the timescale extension installed? (And then check that the scheduler count is untouched)

Reply:
Actually nevermind, I guess you can't call it from a separate database.

Reply:
OK, rephrasing: Can you add a test where, if owner `X` is owner of a DB with timescale installed, then you `drop owned Y`, and check that the scheduler counts don't change.